playbooks/roles/stunnel/templates/stunnel.service.j2

```diff
@@ -0,0 +1,19 @@
+# source https://github.com/liuliang/centos-stunnel-systemd/blob/master/stunnel.service
+[Unit]
+Description=SSL tunnel for network daemons
+After=openvpn@server.target
+
+[Install]
+WantedBy=multi-user.target
+Alias=stunnel.target
+
+[Service]
+Type=forking
+ExecStart=/usr/bin/stunnel {{ stunnel_path }}/stunnel.conf
+ExecStop=/usr/bin/killall -9 stunnel
+
+# Give up if ping don't get an answer
+TimeoutSec=600
+
+Restart=always
+PrivateTmp=false
```

Comment:
Out of curiosity: have you tested stunnel with `PrivateTmp=true`? It looks like `PrivateTmp=false` is present in the linked source unit but without a comment. Maybe there's a good reason for it? To me without justification it seems like a mild red flag since the global `/tmp` is untrustworthy.

Reply:
Good catch, I’ll test this.